test_runner/regress/test_combocid.py

```diff
@@ -0,0 +1,37 @@
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+def test_combocid(neon_env_builder: NeonEnvBuilder):
```

Comment:
@hlinnaka wrote

> I added a test that fails without the fixes from this PR, and passes with them. It only covers the DELETE case, but it should be straightforward to exercise the other cases with a copy-paste of this test.

but I don't see any Heikki's commits in the branch. And to the best of my memory this test looks pretty much the same as it looked a week ago

Are we sure that Heikki's test really got into branch? And/or wasn't removed by some force-push?

Reply:
I did push it, but I think you overwrote it by force-pushing. I'll add it again

Reply:
for the reference, it was this commit: https://github.com/neondatabase/neon/commit/f19107ee9. But now it's detached because of the force-push.

Reply:
> I did push it, but I think you overwrote it by force-pushing

Not me, I was just passing by and following the discussion and RP changes, so was surprised by not seeing a new test anymore :) Thank you for looking into it!

Reply:
Sorry, I did incorrect rebase.